spec/rubocop/cop/naming/constant_name_spec.rb

```diff
@@ -12,6 +12,14 @@
 
   it 'registers an offense for camel case in const name' \
      'when using frozen object assignment' do
```

Comment:
It looks like the `object` and `range` descriptions got switched around.

Reply:
Good catch! 🙂